GitUI/CommandsDialogs/FormCleanupRepository.Designer.cs

```diff
@@ -28,231 +28,238 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
```

Comment:
Looks like you will have to fight with @scottsumrall for who gets the PR merged first (https://github.com/gitextensions/gitextensions/pull/11102), and who has to rebase. The rebase of the designer file is never pretty.

Reply:
Should be easier to merge this first, but the non essential changes should be reverted.